tests/server/member/member_test.go

```diff
@@ -151,7 +152,9 @@ func TestLeaderPriority(t *testing.T) {
 	re := require.New(t)
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	cluster, err := tests.NewTestCluster(ctx, 3)
+	cluster, err := tests.NewTestCluster(ctx, 3, func(conf *config.Config, serverName string) {
+		conf.LeaderPriorityCheckInterval = typeutil.NewDuration(time.Second)
```

Comment:
Will we add similar config to speed up other tests?

Reply:
not see same tests which rely on this config 🤔